### PR TITLE
Upgrade `effector-react` readme to `useUnit`

### PR DESCRIPTION
**Pull request by AlexandrHoroshih**
`useStore` -> `useUnit`

### CODE REVIEW

File: packages/effector-react/README.md [hunk 1]
```diff
@@ -19,29 +19,27 @@ yarn add effector effector-react
...
-})
+const $size = $text.map(text => text.length)
 
 const Form = () => {
-  const {text, size} = useStore($form)
+  const {text, size} = useUnit({
+    text: $text,
+    size: $size,
+  })
+  const [handleTextChange] = useUnit([inputText])
```
Comment (sergeysova) [inline packages/effector-react/README.md:36]: Why not just `useUnit(inputText)` ?

Comment (AlexandrHoroshih) [reply packages/effector-react/README.md:36]: I wanted to show that different kinds of overloads are supported 🤔

Comment (sergeysova) [reply packages/effector-react/README.md]: I think we have explanation in the docs, but readme is just and example, isn't it?

**Comment by zerobias** [comment]
Thanks!

**Pull request merged**